src/test/java/com/google/devtools/build/lib/query2/cquery/FilesOutputFormatterCallbackTest.java

```diff
@@ -134,21 +135,31 @@ private List<String> getOutput(String queryExpression, List<String> outputGroups
   @Test
   public void basicQuery_defaultOutputGroup() throws Exception {
     List<String> output = getOutput("//pkg:all", ImmutableList.of());
-    assertContainsExactlyWithBinDirPrefix(
-        output, "pkg/main_default_file", "pkg/other_default_file");
+    var sourceAndGeneratedFiles = output.stream()
+        .collect(Collectors.<String>partitioningBy(path -> path.startsWith("bazel-out/")));
```

Comment:
Can we get the equivalent without hard-coding "bazel-out"? I'm not sure if this path is exposed elsewhere as some test constant. Otherwise we could find some output path / artifact and parse its segments. 

I'm pretty sure this literal is causing the conflicts.

Reply:
I went with a regex that should be good enough for this test.

Reply:
Thanks. @sgowroji can we re-import the latest version of this PR?

Reply:
Okay!